### ISSUE TITLE
[BUG] Element that contains certain text cannot be clicked on Chromium (on Playwright v1.1.1)

### ISSUE THREAD

**Issue by jinjor** [microsoft/playwright#2786]
**Context:**
- Playwright Version: 1.1.1
- Operating System: Mac
- Node version: v12.13.0
- Browser: Chromium

**Code Snippet**

`page.click(selector)`

**Describe the bug**

<img src="https://user-images.githubusercontent.com/2568148/86208241-1be4c280-bbab-11ea-89f5-a4d39ae4edb1.png" width="300">

In the menu items above, only the "Shutdown" cannot be clicked. It causes the infinite loop. Strangely, changing the text content fixes it. This only happens on Chromium. Firefox and Webkit work well.

I guess that the behavior changes depending on the width of text.

|text|text width|works|
|:--|:--|:--|
|"Shutdown"|72.98px| ❌ |
|"Shutdown "|72.98px| ❌ |
|" Shutdown"|72.98px| ❌ |
|"shutdown"|71.19px| ✅  |
|"Shut down"|76.5px| ✅ |
|"Shutdow"|63.78px| ✅ |
|"Shutdown?"|80.22px| ✅ |

However, the element clicked here is the container, not the text itself.

<img src="https://user-images.githubusercontent.com/2568148/86208428-7aaa3c00-bbab-11ea-94cf-5b75312e2cd5.png" width="">

<img src="https://user-images.githubusercontent.com/2568148/86209370-6c5d1f80-bbad-11ea-8c7f-bbd028174046.png" width="300">

**Other information**

- Changing the font-family fixes it too (it changes the text width too)
- Changing the item order does not fix it
- Changing padding does not fix it
- Changing the icon does not fix it
- I cannot find overlapped elements
- option is `{ timeout: 5000, state: 'visible' }`
- "mousedown" event is used for some reasons

**Log**
```
  pw:api   waiting for selector "#device-list-operation-item-shutdown" +0ms
  pw:api     selector resolved to visible <div class="dropdown-item" id="device-list-operation-…></div> +4ms
  pw:api   attempting page.click action +0ms
  pw:api     waiting for element to be displayed, enabled and not moving +0ms
  pw:api     element is displayed and does not move +27ms
  pw:api     scrolling into view if needed +0ms
  pw:api     done scrolling +1ms
  pw:api     checking that element receives pointer events at (697.09,485) +0ms
  pw:api     element does not receive pointer events +2ms
  pw:api   retrying page.click action +0ms
  pw:api     waiting for element to be displayed, enabled and not moving +0ms
  pw:api     element is displayed and does not move +31ms
  pw:api     scrolling into view if needed +0ms
  pw:api     done scrolling +0ms
  pw:api     checking that element receives pointer events at (697.09,485) +1ms
  pw:api     element does not receive pointer events +2ms
  pw:api   retrying page.click action +0ms
  pw:api     waiting for element to be displayed, enabled and not moving +0ms
  pw:api     element is displayed and does not move +30ms
  pw:api     scrolling into view if needed +0ms
  pw:api     done scrolling +0ms
  pw:api     checking that element receives pointer events at (697.09,485) +1ms
  pw:api     element does not receive pointer events +2ms
  pw:api   retrying page.click action +0ms
  pw:api     waiting for element to be displayed, enabled and not moving +0ms
  pw:api     element is displayed and does not move +31ms
  pw:api     scrolling into view if needed +0ms
  pw:api     done scrolling +1ms
  pw:api     checking that element receives pointer events at (697.09,485) +0ms
  pw:api     element does not receive pointer events +8ms
  pw:api   retrying page.click action +0ms
  pw:api     waiting for element to be displayed, enabled and not moving +0ms
  pw:api     element is displayed and does not move +24ms
  pw:api     scrolling into view if needed +0ms
  pw:api     done scrolling +0ms
  pw:api     checking that element receives pointer events at (697.09,485) +1ms
  pw:api     element does not receive pointer events +2ms

...(loop)

```

**Comment by jinjor** [comment]
**Update:**
I found that v1.1.1 with an older chromium (r764964) works. So this should be a chromium issue.

**Comment by jinjor** [comment]
**Update2:**
Wrapping the text with `<span>` also worked.
<img src="https://user-images.githubusercontent.com/2568148/86221604-0fb73000-bbc0-11ea-8377-0639fd72bedd.png" width="300">

**Comment by pavelfeldman** [comment]
A little tricky to figure it out without knowing where click listeners are registered and how text is styled / laid out. Could it be that the context menu disappears on mouse move? Did you try running it with `headless: false`?

**Comment by jinjor** [comment]
> A little tricky to figure it out without knowing where click listeners are registered and how text is styled / laid out. Could it be that the context menu disappears on mouse move?

- This menu opens with clicking button and closes with clicking anywhere on screen. 
- No interaction with mouse move, no CSS rule like `:hover`.
- The event listener is attached to each container element of menu item (see above). Containers have no gaps and overlaps.
  - (update) It listens "mousedown" event but changing it to "click" did not change the result.

> Did you try running it with headless: false?

I tried with both `headless: false` and `headless: true`. Menu is open while the infinite loop.

**Comment by jinjor** [comment]
I confirmed this problem has been fixed on v1.2.0, so I close this 🙂
